configs/codestyles/ProlificAndroid.xml

```diff
@@ -107,13 +107,10 @@
     <option name="PACKAGES_TO_USE_STAR_IMPORTS">
       <value />
     </option>
-    <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="999" />
-    <option name="NAME_COUNT_TO_USE_STAR_IMPORT_FOR_MEMBERS" value="999" />
+    <option name="NAME_COUNT_TO_USE_STAR_IMPORT" value="2147483647" />
```

Comment:
Is this going to be enough Oo?

Reply:
This is MAX_INT to force the developer to never use wildcard import